Apps/ValidationTests/Win32/App.cpp

```diff
@@ -261,12 +255,11 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
         }
         case WM_SIZE:
         {
-            if (runtime != nullptr) {
+            if (graphics != nullptr)
+            {
                 size_t width = static_cast<size_t>(LOWORD(lParam));
                 size_t height = static_cast<size_t>(HIWORD(lParam));
-                runtime->Dispatch([width, height](Napi::Env env) {
-                    Babylon::Plugins::NativeWindow::UpdateSize(env, width, height);
-                });
+                graphics->UpdateSize(width, height);
             }
```

Comment:
This seems wrong. It appears to be necessary for the validation tests -- I checked -- but it seems wrong. Is it expected that the validation tests should be allowed to resize?

Reply:
Nop, the VT runs with a fixed window size of 600x400. no resize.